docs/api/components/schemas.yaml

```diff
@@ -1884,6 +1884,11 @@ Resync-status:
     docs_processed:
       description: The amount of docs that have been processed so far in the resync operation.
       type: integer
+    collections_processing:
```

Comment:
maybe add an example of scopename.collectionname if that's the syntax that this takes?